src/app.py

```diff
@@ -28,9 +28,9 @@
 DATA_LOCATION = "/data"
 ENVIROMENT = "development"
 LOCALSTACK_ENDPOINT = "http://localstack:4566"
-SOURCE_BUCKET_NAME = "biomage-source-development"
-PROCESSED_MATRIX_BUCKET_NAME = "processed-matrix-development"
-CELL_SETS_BUCKET_NAME = "cell-sets-development"
+SOURCE_BUCKET_NAME = "biomage-source-development-242905224710"
+PROCESSED_MATRIX_BUCKET_NAME = "processed-matrix-development-242905224710"
+CELL_SETS_BUCKET_NAME = "cell-sets-development-242905224710"
```

Comment:
Can you not hardcode this? Maybe use a variable with default? e.g. `(AWS_ACCOUNT_ID = env.get("AWS_ACCOUNT_ID", DEFAULT_ACCOUNT_ID)`

Reply:
Not hardcoding the account id is for a future ticket. For now they are hardcoded like this.